Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks.yml

```diff
@@ -2200,3 +2200,19 @@
               Quantity: 30
     - type: Sprite
       sprite: Objects/Consumable/Drinks/kvass.rsi
+
+- type: entity
+  parent: DrinkGlassBase
+  id: DrinkDavidMartinez
+  name: "The Martinez" glass
+  description: The edgerunner legend.  Remembered by a drink, Forgotten by a drunk.
+  components:
+    - type: SolutionContainerManager
+      solutions:
+        drink:
+          maxVol: 30
+          reagents:
+            - ReagentId: DavidMartinez
```

Comment:
you changed reagent to TheMartinez so replace this and rename other stuff to follow

Reply:
yea i saw that and changed it.